Observer/Config/Save.php

```diff
@@ -155,13 +168,35 @@ private function resetStoreCredentials($scopeId = null, $scope = ScopeInterface:
      * @param array <string> $changedPaths
      * @return bool
      */
-    public function isYotpoKeysChanged($changedPaths = [])
+    public function isYotpoSettingsChanged($changedPaths = [])
     {
         $yotpoKeyPaths = ['app_key', 'secret', 'yotpo_active'];
+        $commonPaths = $this->getChangedYotpoPaths($changedPaths, $yotpoKeyPaths);
+        return (bool)$commonPaths;
+    }
+
+    /**
+     * @param array <string> $changedPaths
+     * @return bool
+     */
+    public function isYotpoKeysChanged($changedPaths = [])
```

Comment:
The name is wrong, please modify.
```suggestion
    public function isYotpoAppKeyChanged($changedPaths = [])
```

Reply:
This is done. Will update the PR soon